index.html

```diff
@@ -1,31 +1,32 @@
 <!doctype html>
 <html>
-  <head>
-    <link rel='stylesheet' href='styles.css'>
-  </head>
-  <body>
-    <header>
-      <h1>The Header</h1>
-    </header>
-    <aside>
-      <h2>A Side Bar</h2>
-    </aside>
-    <div class='main'>
-      <h2>The Main Content</h2>
-      <div class='left'>
-	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
-      </div>
-      <div class='right'>
-	This is the right side
-      </div>
+<head>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <title>Home</title>
+</head>
+<body>
+  <header>
+    <h1>The Header</h1>
+  </header>
+  <aside>
+    <h2>A Side Bar</h2>
+  </aside>
+  <div class='main'>
```

Comment:
for question 2 we wanted to see something non semantic like `<div class='main'>` changed to `<main>`